src/or/entrynodes.c

```diff
@@ -142,6 +142,9 @@
 #include "or/node_st.h"
 #include "or/origin_circuit_st.h"
 
+#define REMOVE_UNLISTED_GUARDS_AFTER \
```

Comment:
Hmm, not sure if I like this macro doing multiplications and casting to time_t. Suggestion: Let's modify `get_remove_unlisted_guards_after_days()` to return exactly what we want, and let's do the multiplication in the log of `sampled_guards_prune_obsolete_entries()`. What do you think?

Reply:
In 64685c37b670cd539848377f8c6febc22fd75cb4 I made a function that returns the exact number we need here.